.github/workflows/ci.yml

```diff
@@ -382,6 +390,18 @@ jobs:
           $CC --version
           $CXX --version
 
+    - name: Setup MATLAB (if SWIGLANG == matlab)
+      if: contains(matrix.SWIGLANG, 'matlab')
+      uses: matlab-actions/setup-matlab@v1
+      with:
+        release: ${{ matrix.VER }}
+
+    # workaround for https://github.com/robotology/robotology-superbuild/issues/64
+    - name: Do not use MATLAB's stdc++ to avoid incompatibilities with other libraries
+      if: contains(matrix.SWIGLANG, 'matlab')
+      run:
+          echo 'LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6' >> $GITHUB_ENV
```

Comment:
is this required for the SWIG tests? If so, I wonder if we shouldn't compile the test-suite only with MATLAB supported versions of the compiler which should remove this requirement (although that might be hard to do)

Reply:
Sorry for the late reply. In theory MATLAB after R2020b support 20.04, but the test do not pass in Ubuntu 20.04 if I remove this workaround (perhaps some GitHub Actions specific modifications?). However, by setting the os to ubuntu-18.04 I can get all the test to pass without any workaround.

Reply:
ok. I guess this is because `gcc` in 18.04 is older than in 20.04, and is therefore supported by MATLAB. I'm happy with this change (it's not our business to fix MATLAB...)